web/docs/auth/auth-hooks.md

```diff
@@ -220,9 +226,9 @@ export const onAfterSignup = async ({
   console.log('number of users after', count)
   console.log('user object', user)
 
-  // If this is an OAuth signup, we have the access token and uniqueRequestId
+  // If this is an OAuth signup, we have the OAuth tokens and uniqueRequestId
```

Comment:
Address the user directly plus "the" before `uniqueRequestId`. Same applies below.